packages/ui/src/stories/Markdown.stories.tsx

```
@@ -153,4 +162,76 @@ export default {
 
-export const Markdown: StoryObj<typeof Component> = {}
+// Story with interactive checkboxes
+const InteractiveCheckboxesTemplate = (args: any) => {
+    const [markdownState, setMarkdownState] = useState(args.children)
+
+    return (
+        <div>
+            <h1>Markdown with Interactive Checkboxes:</h1>
+            <Component
+                {...args}
+                children={markdownState}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)